test/presubmit.sh

```diff
@@ -18,8 +18,13 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-echo "+++ Building cmrel tool"
-go build -o cmrel ./cmd/cmrel
+if [ -z ${1+x} ]; then
+	echo "+++ Building cmrel tool"
+	go build -o ./cmrel ./cmd/cmrel
+	CMREL=./cmrel
+else
+	CMREL=$1
+fi
```

Comment:
note: this will change to basically be `CMREL=$1` after the [test job](https://github.com/jetstack/testing/blob/master/config/jobs/cert-manager/release/cert-manager-release-presubmits.yaml) is changed to use the Makefile after this merges.